File: JS-Programming-Basics/HomeWorks-And-Labs/01-FirstStepsInCodin-Lab/09.YardGreening/09. Yard Greening.js
function input(input) {
    let areaForGreening = input[0];
    let priceForOneSqMeter = 7.61;
    let discount = 0.18;
    let totalPriceForGreening = areaForGreening * priceForOneSqMeter;
    let discountPrice = discount * totalPriceForGreening;
    let totalPrice = totalPriceForGreening - discountPrice;

    console.log(`The final price is: ${totalPrice} lv.`);
    console.log(`The discount is: ${discountPrice} lv.`)


}
input (["550"]);

File: Python-Programming-Basics/HomeWorks-And-Labs/01.FirstSteps_In_Coding-lab/09.Yard_Greening.py
total_sq_m = float(input())

price_sq_m = 7.61
before_discount_total_prc = total_sq_m * price_sq_m
discount = before_discount_total_prc * 0.18
after_discount_total_prc = before_discount_total_prc - discount

print(f'The final price is: {after_discount_total_prc} lv.')
print(f'The discount is: {discount} lv.')